web-client/src/views/Messages/MessagesIndividualCompleted.jsx

```diff
@@ -1,21 +1,80 @@
+import {
+  ALPHABETICALLY_ASCENDING,
+  ALPHABETICALLY_DESCENDING,
+  CHRONOLOGICALLY_ASCENDING,
+  CHRONOLOGICALLY_DESCENDING,
+} from './sortConstants';
```

Comment:
❓ Why were these added to a `sortConstants` over adding them to the existing applicationContext constants?

Reply:
No particular reason. We have migrated those items to application context.